src/features/events/store.ts

```diff
@@ -109,7 +109,10 @@ const eventsSlice = createSlice({
       action: PayloadAction<[number, ZetkinEventParticipant]>
     ) => {
       const [eventId, participant] = action.payload;
-      state.participantsByEventId[eventId].items
+      state.participantsByEventId[eventId].items;
+      state.participantsByEventId[eventId].items = state.participantsByEventId[
```

Comment:
Did it not work to just push? [The `immer` documentation](https://immerjs.github.io/immer/update-patterns/) seems to state that it should (note that the `produce()` part is irrelevant for us, it's handled by redux toolkit).

Reply:
Yes! It's working! Nice to have less lines of code to do this :)